Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml

```diff
@@ -671,10 +671,10 @@
     price: 100
 
 - type: entity
-  name: experimental C.H.I.M.P. handcannon
-  parent: [WeaponPistolCHIMP, BaseSyndicateContraband]
+  name: C.H.I.M.P. handcannon
+  parent: [WeaponPistolCHIMP, BaseScienceContraband]
```

Comment:
Its not science contraband, its syndicate contraband.

Reply:
It’s meant to be a stealth item now, so having it be syndicate contraband would defeat the point